js/templates/modals/settings/page.html

```diff
@@ -79,10 +79,11 @@ <h2 class="h3 txUnb"><%= ob.polyT('settings.pageTab') %></h2>
     <div class="flexRow">
       <div class="col3">
         <label><%= ob.polyT('settings.avatar') %></label>
+        <div class="clrT2 txBase"><%= ob.polyT('settings.loadAvatarHelp', {minWidth: ob.avatarMinWidth, minHeight: ob.avatarMinHeight}) %></div>
       </div>
       <div class="col9">
         <div class="flexRow gutterH" id="avatarCropper">
-          <div class="contentBox avatarPreview clrP clrBr flexNoShrink js-avatarPreview"></div>
+          <div class="contentBox avatarPreview clrP flexNoShrink js-avatarPreview"></div>
```

Comment:
If we take the border color off here, and you upload a photo with a white background, you can't see the circle any more (if you have the primary color still set to the default). 

![image](https://cloud.githubusercontent.com/assets/1584275/21276226/6c57072c-c39e-11e6-9a26-048a15ffc50a.png)